website/docs/docs/dbt-versions/core-upgrade/05-upgrading-to-v1.10.md

```diff
@@ -49,6 +49,15 @@ You can read more about each of these behavior changes in the following links:
 
 - (Introduced, disabled by default) [`validate_macro_args`](/reference/global-configs/behavior-changes#macro-argument-validation). If the flag is set to `True`, dbt will raise a warning if the argument `type` names you've added in your macro YAMLs don't match the argument names in your macro or if the argument types aren't valid according to the [supported types](/reference/resource-properties/arguments#supported-types).
 
+### Deprecation warnings as errors
+
+You can now use the [`--warn-error`](/reference/global-configs/warnings) flag to set deprecation warnings as errors. Using [`--warn-error-option`](/reference/global-configs/warnings#use---warn-error-options-for-targeted-warnings), you can set targeted feature deprecation warnings as errors in combination with other settings to cover scenarios such as:
+
+- Set only deprecation warnings as errors with `--warn-error-options="{'error': ['deprecations']}"`
+- Set a specific deprecation warning as an error while silencing all other deprecation warnings with `--warn-error-options="{'error': ['OneSpecificDeprecation'], 'silence': ['deprecations']}"`
```

Comment:
i think we should give an example of a real deprecation name _not_ "OneSpecificDeprecation"

Reply:
How about `PackageRedirectDeprecation`?